view/src/components/Callstack.jsx

```diff
@@ -0,0 +1,264 @@
+// Call stack visualisation.
+// Author: Lukas Petr
+
+import ListGroup from 'react-bootstrap/ListGroup';
+import { PropTypes } from 'prop-types';
+import { CallstackPropTypes, DefinitionsPropTypes } from '../PropTypesValues';
+
+/**
+ * Component for visualisation of call stack.
+ * @param {Object} props
+ * @param {string} props.compFunName - First (compared) function name.
+ * @param {string} props.selectedFunction - Name of function which is selected
+ * and should be highlighted in call stack.
+ * @param {Function} props.onSelect - Callback function accepting a function
+ * name which was chosen to be shown.
+ * @returns Returns call stack component.
+ */
+export default function Callstack({
+  compFunName,
+  oldCallStack,
+  newCallStack,
+  selectedFunction,
+  onSelect,
+  definitions,
+}) {
+  function getNewName(oldName) {
+    if (oldName in definitions) {
+      if ('name' in definitions[oldName].new) {
+        return definitions[oldName].new.name;
+      }
+    }
+    return oldName;
```

Comment:
I used [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining), hopefully it will make it even more cleaner.